anni-provider/Cargo.toml

```diff
@@ -9,17 +9,17 @@ repository = "https://github.com/ProjectAnni/anni"
 license = "Apache-2.0"
 
 [dependencies]
-tokio = { version = "1", features = ["time"] }
+tokio = { version = "1", features = ["time", "fs", "rt"] }
```

Comment:
why fs and rt are added?

Reply:
otherwise the compilation without some features will fail due to the use of `tokio::fs` and  `tokio::spawn`

Reply:
Add `tokio/xxxx` to corresponding features? But why the compilation did not fail before(?

Reply:
> But why the compilation did not fail before(?

IMO other dependencies enabled these features but they are now optional.

Reply:
I think fs is reasonable, but rt seems that we're asking a library to depend on a runtime, which is confusing to me.

(Might not be correct. I'm not familiar with this part.)

Reply:
Digged deeper, and the `rt` feature looks reasonable to me now.